client/src/components/AlumniProfile.js

```diff
@@ -286,28 +246,149 @@ export default class AlumniProfile extends Component {
                     size="medium"
                     src={details.imageURL}
                 />
+                {
+                !isViewOnly ?
+                    <>
+                        <Button
+                        style={{'width': '30%', 'align-self': 'center', 'margin': '5px 0 5px 0'}}
+                        basic
+                        color="blue"
+                        onClick={this.openImageModal}
+                        >
+                            Update Image
+                        </Button>
+                        <ImageUpdateModal
+                            modalOpen={this.state.imageModalOpen}
+                            id={details._id}
+                            isAlumni={true}
+                            closeModal={this.closeImageModal}
+                        />
+                    </>
+                    : null
+                }
```

Comment:
Design comment - this might look better as a Dimmer from semantic UI

Here's a custom snippet:
export default class DimmerExampleInverted extends Component {
  state = {}

  handleShow = () => this.setState({ active: true })
  handleHide = () => this.setState({ active: false })

  render() {
    const { active } = this.state

    return (
      <div>
         <Dimmer.Dimmable as={Image}
          dimmed={active}
          onMouseEnter={this.handleShow}
          onMouseLeave={this.handleHide} 
        >
          <Dimmer active={active} inverted verticalAlign='bottom'>
            <Button verticalAlign='bottom' primary>
              Update Image
            </Button>
          </Dimmer>
            <Image rounded size='medium' src='https://techcrunch.com/wp-content/uploads/2010/07/github-logo.png?w=512' />
        </Dimmer.Dimmable>

        <Button.Group>
          <Button icon='plus' onClick={this.handleShow} />
          <Button icon='minus' onClick={this.handleHide} />
        </Button.Group>
      </div>
    )
  }
}

Apologies for the rough formatting, but it looks like this:
![image](https://user-images.githubusercontent.com/54961598/85929883-3efb3380-b886-11ea-87ed-814b76a16dd4.png)
![image](https://user-images.githubusercontent.com/54961598/85929890-47536e80-b886-11ea-8526-ccd97d00e84e.png)

Reply:
Oooh fancy, done!